docs/sphinx/source/whatsnew/v0.9.4.rst

```diff
@@ -13,6 +13,7 @@ Enhancements
 
 Bug fixes
 ~~~~~~~~~
+* Multiple issues fixed that were reported by LGTM analysis.
```

Comment:
```suggestion
* Multiple code style issues fixed that were reported by LGTM analysis. (:issue:`1275`, :pull:`1559`)
```

Reply:
Also, since no functionality is changing here, I don't think it makes sense to call this a bugfix PR.  I suppose I'd put it under the `Enhancement` section?